electrum/gui/qt/staking_detail_tx_window.py

```diff
@@ -318,21 +327,23 @@ def insert_data(self, vbox):
 
         hbox_period = QHBoxLayout()
         period_lab = QLabel('Staking period:')
-        period_lab_data = QLabel(f"{self.data.staking_info.staking_period/144} Days")
+        period = int(self.data.staking_info.staking_period / 144)
```

Comment:
we definitely need some common place for 'blocks to days' method in staking/utils.py